src/pages/index.js

```diff
@@ -14,81 +16,190 @@ const addCardAddButton = document.querySelector(selectors.cardAddButton);
-newCardPopup.setEventListeners();
+});
 
-//USER INFO
+// User Info instance
 const userInfo = new UserInfo({
-    nameSelector: selectors.profileHeadingElement,
-    descriptionSelector: selectors.profileDescriptionElement
+  nameSelector: selectors.profileHeadingElement,
+  descriptionSelector: selectors.profileDescriptionElement,
+  avatarSelector: selectors.avatarImageElement,
 });
 
-//USER PROFILE
-const updateProfilePopup = new PopupWithForm(
-  selectors.profilePopup,
-  handleProfileSubmit
-);
+// Initial card section
+let cardSection;
+api
+  .initialPageLoad()
+  .then(([cards, user]) => {
+    cardSection = new Section(
+      {
+        items: cards,
+        renderer: (item) => {
+          const card = renderCard(item);
+          cardSection.addItem(card);
+        },
+      },
+      selectors.cardSection
+    );
+    cardSection.renderItems();
+    userInfo.setUserInfo(user);
+    userInfo.setUserAvatar(user.avatar);
+  })
+  .catch((err) => {
+    console.error(`Error: ${err}`);
+  });
+
+// Profile popup
+const updateProfilePopup = new PopupWithForm(selectors.profilePopup, handleProfileSubmit);
 updateProfilePopup.setEventListeners();
 
+// New card popup
+const newCardPopup = new PopupWithForm(selectors.cardPopup, handleAddCardFormSubmit);
+newCardPopup.setEventListeners();
+
+// Delete card popup
+const deletePopup = new PopupConfirm(selectors.deletePopup, handleDeleteCard);
+deletePopup.setEventListeners();
 
-function renderCard(item) {
-  const card = new Card(item, selectors.cardTemplate, handleImageClick);
-  return card.generateCard();
-}
+// Edit avatar popup
+const avatarPopup = new PopupWithForm(selectors.avatarPopup, handleEditAvatar);
+avatarPopup.setEventListeners();
 
-// ! ||--------------------------------------------------------------------------------||
-// ! ||                                 Event Listeners                                ||
-// ! ||--------------------------------------------------------------------------------||
+// Event Listeners
 addCardAddButton.addEventListener("click", () => {
   newCardPopup.open();
   cardFormValidator.toggleButtonState();
 });
 
 updateProfileButton.addEventListener("click", () => {
   const userData = userInfo.getUserInfo();
-  profileHeadingInput.value = userData.userName;
-  profileDescriptionInput.value = userData.userDescription;
+  profileHeadingInput.value = userData.name;
+  profileDescriptionInput.value = userData.about;
   editProfileFormValidator.resetValidation();
   updateProfilePopup.open();
 });
 
-// ! ||--------------------------------------------------------------------------------||
-// ! ||                                 Event Handlers                                 ||
-// ! ||--------------------------------------------------------------------------------||
-function handleAddCardFormSubmit(data) {
-  const card = renderCard(data);
-  cardSection.addItem(card);
+updateAvatarButton.addEventListener("click", () => {
+  avatarPopup.open();
```

Comment:
Please, call `toggleButtonState` for the avatar form to disable the button if the input is empty